test/e2e/resize_nodes.go

```diff
@@ -361,10 +361,15 @@ func performTemporaryNetworkFailure(c *client.Client, ns, rcName string, replica
 		}
 	}()
 
+	Logf("Waiting for node %s to be ready", node.Name)
```

Comment:
OK NVM actually read through in my editor (I blame Github but it's my fault)—will re-comment soon...